src/components/Header/Header.jsx

```diff
@@ -1,20 +1,44 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { useLocation, Link } from 'react-router-dom';
+
+import './Header.scss';
 
 function Header() {
+  const [hash, setHash] = useState(null);
+  const currentHash = useLocation().pathname;
+
+  useEffect(() => {
+    setHash(currentHash);
+  }, [currentHash]);
+
   return (
     <div className="header">
       <Link className="header__logo" to="/">WeatherApp</Link>
       <nav className="menu">
         <ul className="menu__list">
           <li className="menu__list-item">
-            <Link className="menu__list-link" to="/">Home</Link>
+            <Link
+              className={`menu__list-link ${hash === '/' ? 'active' : ''}`}
```

Comment:
as you don't use css modules, what if you'll have "active" class name in other components? please use BEM completely or don't use it at all